ui/src/app/components/gke-app-configuration-panel.tsx

```diff
@@ -23,7 +23,7 @@ import { CreateRStudio } from './gke-app-configuration-panels/create-rstudio';
 import { CreateSAS } from './gke-app-configuration-panels/create-sas';
 
 type InjectedProps =
-  | 'app'
+  | 'userApps'
```

Comment:
we now pass in the list of the user's apps (in the workspace) to this component, because we need to know about the statuses of other apps